.circleci/config.yml

```diff
@@ -16,6 +13,16 @@ jobs:
     steps:
       - checkout
       - run: gruntwork-install --binary-name 'terratest_log_parser' --repo 'https://github.com/gruntwork-io/terratest' --tag 'v0.13.8'
```

Comment:
NIT: While we are here, we should bump this to `v0.29.0`.

Reply:
Done: https://github.com/gruntwork-io/terragrunt/pull/1357/commits/c2185a629981636ae9238d18cf9516ac9d2ad3b7